harvester/harvester-installer
# Replace luet-makeiso with elemental build-iso.

**Problem:**

We need to migrate from `luet-makeiso` to `elemental build-iso`.

**Solution:**

Add `elemental` base image for elemental binary and `rsync` package for elemental `build-iso` command execution.

**Related Issue:**
https://github.com/harvester/harvester/issues/2204

**Test plan:**

Boot:
1. Build an ISO from `elemental-build` branch (both `harvester` and `harvester-installer` repo);
2. Try to boot up using this ISO in UEFI environment, it should boot up without problem;
3. Try to boot up using this ISO in legacy BIOS environment too.

Upgrade:
1. Build an ISO from `elemental-build` branch (both `harvester` and `harvester-installer` repo);
2. Install a clean environment using v1.1.2;
3. Create a http server to serve the ISO file;
4. Create a version object in Harvester and let Harvester upgrade to the ISO just built;
5. Check if Harvester upgraded successfully.

**Additional Information:**
We might need to create another issue to add legacy BIOS boot capability if it's necessary.
